Adafruit_FXAS21002C.cpp

```diff
@@ -120,91 +92,23 @@ Adafruit_FXAS21002C::Adafruit_FXAS21002C(int32_t sensorID, byte addr) {
 /*!
     @brief  Setup the HW
 
-    @param  rng
-            The range to set for the gyroscope, based on gyroRange_t
+    @param addr The I2C address of the sensor.
+    @param wire Pointer to Wire instance
 
     @return True if the device was successfully initialized, otherwise false.
 */
 /**************************************************************************/
-bool Adafruit_FXAS21002C::begin(gyroRange_t rng) {
-  /* Enable I2C */
-  Wire.begin();
+bool Adafruit_FXAS21002C::begin(uint8_t addr, TwoWire *wire) {
 
-  /* Set the range the an appropriate value */
-  _range = rng;
-
-  /* Clear the raw sensor data */
-  raw.x = 0;
-  raw.y = 0;
-  raw.z = 0;
-
-  /* Make sure we have the correct chip ID since this checks
-     for correct address and that the IC is properly connected */
-  uint8_t id = read8(GYRO_REGISTER_WHO_AM_I);
```

Comment:
we should still check the ID register too

Reply:
yep. thats done in `begin()`.

Reply:
oh duh